Pipfile

```diff
@@ -32,8 +32,6 @@ werkzeug = "*"
 fakeredis = "*"
 email-validator = "*"
 pyjwt = "*"
-isort = "*"
```

Comment:
It looks like you might have forgotten to regenerate the lock file. Sorry for slow re-review, an ill daughter and some time off has delayed things